.vscode/cspell.json

```diff
@@ -101,6 +101,7 @@
     "sdk/translation/azure-ai-translation-text/tests/test_break_sentence_async.py",
     "sdk/translation/azure-ai-translation-text/tests/test_translation_async.py",
     "sdk/translation/azure-ai-translation-text/tests/test_transliteration_async.py",
+	"sdk/translation/azure-ai-translation-text/azure/ai/translation/text/_model_base.py",
```

Comment:
I think this is tab vs spaces issue. Please fix it.

Reply:
Fixed and pushed.